### PR TITLE
Fix test ownership lint

### PR DESCRIPTION
**Pull request by janeyx99**
I noticed after creating https://github.com/pytorch/pytorch/issues/71553 that the test ownership lint was not working properly.

This fixes my egregious mistake and fixes the broken lints.

### CODE REVIEW

File: test/fx2trt/converters/acc_op/test_chunk.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Owner(s): ["oncall: fx"]
```
Comment (janeyx99) [inline test/fx2trt/converters/acc_op/test_chunk.py:1]: @wushirong can you confirm this is the proper owner of these?

File: test/test_functionalization.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# Owner(s): ["module: codegen"]
```
Comment (janeyx99) [inline test/test_functionalization.py:1]: @bdhirsh can you confirm this owner?

File: test/test_monitor.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# Owner(s): ["module: unknown"]
```
Comment (janeyx99) [inline test/test_monitor.py:1]: @d4l3k is there a current owner to torch.monitor? An owner is typically a team or group of people who have the expertise to investigate concerns and answer questions. Owners are usually encompassed by a `module: blah` or `oncall: blah` label on pytorch.

File: test/test_masked.py [hunk 4]
```diff
@@ -1,3 +1,5 @@
+# Owner(s): ["module: masked operators"]
```
Comment (janeyx99) [inline test/test_masked.py:1]: @pearu Would this be the proper owner?

File: test/test_functionalization.py (hunk 2 above)
Comment (bdhirsh) [reply test/test_functionalization.py:1]: yeah codegen seems fine!

File: test/test_monitor.py (hunk 3 above)
Comment (d4l3k) [reply test/test_monitor.py:1]: We have a `pytorch_r2p` oncall which I think should work here?

https://www.internalfb.com/omh/view/pytorch_r2p/frontpage

Comment (janeyx99) [reply test/test_monitor.py:1]: yes! thank you

File: test/test_masked.py (hunk 4 above)
Comment (janeyx99) [reply test/test_masked.py:1]: and if so, do you know who usually looks at `module: masked operators` tagged issues/prs? We may want to add them to https://github.com/pytorch/pytorch/issues/24422

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**